framework/wazuh/core/config/models/base.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel, ConfigDict
+
+class WazuhConfigBaseModel(BaseModel):
+    """Main model for the configuration sections."""
+    model_config = ConfigDict(use_enum_values=True)
```

Comment:
Add EOF line

Reply:
Fixed in c24216b9677fbc2f0325ac805d258b4105f66c3d.